stripper/ze_stalker_ultimate_v4_9.cfg

```diff
@@ -77,6 +77,6 @@ modify:
 	}
 	insert:
 	{
-		"OnTrigger" "point_servercommandCommandsm_setcooldown 550514 805-1"
+		"OnTrigger" "point_servercommandCommandsm_setcooldown flash_activate 805-1"
```

Comment:
What vauff meant was change to using the point_servercommand's targetname, not the item's. For using sm_setcooldown, the command needs to target the hammerid of the pistol (same as set in the entwatch, in this case 550582), not the button's hammerid.

So it should look something like this: `"OnTrigger" "server,Command,sm_setcooldown 550582 80,5,-1"`